packages/blade/src/components/ListView/types.ts

```diff
@@ -40,6 +45,8 @@ type ListViewFilterProps = {
   showQuickFilters?: boolean;
   /**
    * it will show/hide the filters
+   * @deprecated Filters are now always expanded.
```

Comment:
"Filters are now always expanded." okay but 🤔  why does it have to be marked as deprecated?

Reply:
Like does this prop not do anything anymore?